waku/v2/node/storage/migration/migrations_scripts/message/00002_addSenderTimeStamp.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS Message_backup (
```

Comment:
I think this script should only run if `Message` actually exists (i.e. add an `IF EXISTS Message` that determines if script is run or not). 
I might be missing something, but as it stands, migrations will always run if there is a `dbPath` set - whether `store` and `persistMessages` are enabled or not. I think migration scripts for specific tables should only run if those tables were created before.

Reply:
Thanks for identifying the issue! you are right. I have changed the code structure now, the migration is only run when the `persistMessages` is enabled (there is no migration call for the peer store at the moment). 
I am working on updating the SQLite migration scripts in a way that they only get run when the corresponding table exists, however, SQLite is a bit tricky about conditional statement execution and it is more involved than `IF EXISTS Message` It does not allow conditioning the creation of one table to the existence of another table within the same script. Will address it in the following PRs.